src/components/extras/embeddable-page.vue

```diff
@@ -0,0 +1,94 @@
+<!--
```

Comment:
Please use underscores for naming convention not hyphens. Also, the extras folder is reserved for components that are shared between different routes/views. This component doesn't fit, make a new folder kiosks in components.

Reply:
Fixed both of these issues